pkg/query-service/app/clickhouseReader/reader.go

```diff
@@ -2593,11 +2593,11 @@ func (r *ClickHouseReader) CountErrors(ctx context.Context, queryParams *model.C
 	args := []interface{}{clickhouse.Named("timestampL", strconv.FormatInt(queryParams.Start.UnixNano(), 10)), clickhouse.Named("timestampU", strconv.FormatInt(queryParams.End.UnixNano(), 10))}
 	if len(queryParams.ServiceName) != 0 {
```

Comment:
@makeavish how was this working earlier as I see `serviceName` is now added to `CountErrorsParams` struct

Reply:
It wasn't working. i.e. filters weren't applied at all.
Initially, I started on this but later due to context switch missed this scenario.